index.html

```diff
@@ -2787,9 +2787,10 @@ <h3>JWT and JWS Considerations</h3>
       "https://w3.org/2018/credentials/v1",
       "https://example.com/examples/v1"
     ],
-    "type": ["VerifiableCredential", "ProofOfAgeCredential"],
-    "credentialSubject": {
-      "ageOver": 21
+    "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+    "degree": {
```

Comment:
This VC should have a "credentialSubject" property that contains the "degree" property and an "id" that matches "sub" above.